common/formats/prodos.cpp

```diff
@@ -31,7 +31,8 @@ ProDOSFile::ProDOSFile(char name[16], uint8 type, uint16 tBlk, uint32 eof, uint1
 	, _eof(eof)
 	, _blockPtr(bPtr)
 	, _disk(disk) {
-	strncpy(_name, name, 16);
```

Comment:
As I understand, you wanted to make sure NUL is present. Then:
```C++
Common::strlcpy(_name, name, 16);
```

Reply:
Done